src/sage/schemes/elliptic_curves/period_lattice.py

```diff
@@ -857,9 +857,9 @@ def real_period(self, prec=None, algorithm='sage'):
         - ``prec`` (int or ``None`` (default)) -- real precision in
```

Comment:
```suggestion
        - ``prec`` (integer or ``None`` (default)) -- real precision in
```

Reply:
hmm I see that there's many `int` and many `integer` -- now im less confident in whcih we should use here.

Reply:
I prefer "integer".

Best to take care of in a follow-up just focused on this change and apply it systematically